client/src/components/Ticketing/ticketingmanager/Doorlistpage/doorlist.tsx

```diff
@@ -262,20 +263,16 @@ const DoorList = () => {
           {date && time ? `${date}, ${time}` : `${date}${time}`}
         </h3>
         <div className='bg-white p-5 rounded-xl mt-2 shadow-xl'>
-          {ticketsSold ? (
-            <DataGrid
-              className='bg-white'
-              autoHeight
-              disableSelectionOnClick
-              rows={doorList}
-              columns={columns}
-              pageSize={10}
-              components={{
-                Toolbar: CustomToolbar,
-              }} />
-          ) : (
-            <p className="text-xl font-bold text-red-600">No tickets sold for this show</p>
-          )}
+          <DataGrid
+            className='bg-white'
+            autoHeight
+            disableSelectionOnClick
+            rows={doorList}
+            columns={columns}
+            pageSize={10}
+            components={{
+              Toolbar: CustomToolbar,
+            }} />
```

Comment:
Having the checkbox under the Arrived column be checkable doesn't quite make sense since they are open seats. I would have it grayed out! Otherwise, everything looks good!

Reply:
fixed!